instrumentor/controllers/utils/labels.go

```diff
@@ -0,0 +1,53 @@
+package utils
+
+import (
+	"context"
+
+	"github.com/odigos-io/odigos/common/consts"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+func IsWorkloadInstrumentationEffectiveEnabled(ctx context.Context, kubeClient client.Client, obj client.Object) (bool, error) {
```

Comment:
nit: there are already similar functions in k8sutils https://github.com/odigos-io/odigos/blob/main/k8sutils/pkg/workload/workload.go

consider moving these function there so they can be used by ui / cloud proxy etc.